### PR TITLE
Add auth attributes and change test size for go_default_test

### PR DESCRIPTION
**Pull request by apicl**
#1503 
**Release note**:

```release-note
NONE
```

---
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/1505)

### CODE REVIEW

File: pkg/il/evaluator/BUILD [hunk 1]
```diff
@@ -21,7 +21,7 @@ go_library(
 
 go_test(
     name = "go_default_test",
-    size = "small",
+    size = "medium",
```
Comment (geeknoid) [inline pkg/il/evaluator/BUILD]: Why this change?

**Comment by geeknoid** [comment]
@douglas-reid @sebastienvas 

What happens when we set a test size to medium? Will the lab still run these tests as part of precommit checks?

**Comment by apicl** [comment]
It still runs as part of the pre-push. The only different is that the pre-push will wait a little longer.

**Comment by apicl** [comment]
PTAL

**Comment by douglas-reid** [comment]
@geeknoid `small` -> `medium` shouldn't have any observable impact. moving to `large` or bigger will, based on our configuration, which includes:

```
test --test_size_filters=-large,-enormous
